.github/workflows/scheduled-container-scanning.yml

```diff
@@ -0,0 +1,51 @@
+name: Scheduled docker container scan
+
+on:
+  schedule:
+    - cron: "0 21 * * 1-5"
+  workflow_dispatch:
+
+env:
+  CONTAINER_REGISTRY_URL: 'ghcr.io/infonl'
+  APPLICATION_NAME: 'zaakafhandelcomponent'
+
+permissions:
+  security-events: write
+
+jobs:
+  build:
+    name: Run docker scan
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Pull Docker Image
+        run: docker pull ${{ env.CONTAINER_REGISTRY_URL }}/${{ env.APPLICATION_NAME }}:latest
```

Comment:
Minor thing but a bit nicer to define the Docker image name in a var at the top of this workflow in an `env:` section and re-use it instead of repeating it three times. :-)